components/table/util.ts

```diff
@@ -33,19 +33,26 @@ function getElementFirstTextContent(node: React.ReactElement): string {
   );
 }
 
+/**
+ * Render title by first content
+ *
+ * @param title
+ * @returns
+ */
+export function renderColumnTitleWithFirstContent<RecordType>(title: ColumnTitle<RecordType>) {
+  if (React.isValidElement(title)) {
+    return getElementFirstTextContent(title);
```

Comment:
`getElementFirstTextContent` for check children is not a goodway which is easy to break on `HOC`:
```tsx
const MyTitle = () => 'title';

columns=[{
  title: <MyTitle />,
}]
```